src/bp/ui-admin/src/Pages/Bot/Details.jsx

```diff
@@ -162,33 +191,30 @@ class Bots extends Component {
   handleCategoryChanged = category => this.setState({ category })
 
   handleDefaultLangChanged = lang => {
-    if (!this.state.defaultLanguage) {
-      this.setState({ defaultLanguage: lang.value })
+    if (!this.state.selectedDefaultLang) {
+      this.setState({ selectedDefaultLang: lang })
       return
     }
 
-    if (this.state.defaultLanguage != lang.value) {
+    if (this.state.selectedDefaultLang !== lang) {
       const conf = window.confirm(
-        `Are you sure you want to change the language of your bot from ${this.state.defaultLanguage.toUpperCase()} to ${
+        `Are you sure you want to change the language of your bot from ${this.state.selectedDefaultLang.label} to ${
           lang.label
         }? All of your content elements will be copied, make sure you translate them.`
       )
 
       if (conf) {
-        this.setState({ defaultLanguage: lang.value })
+        this.setState({ selectedDefaultLang: lang })
       }
     }
   }
 
   handleLanguagesChanged = langs => {
-    this.setState({ languages: langs.map(l => l.value) })
+    this.setState({ selectedLanguages: langs })
   }
 
   handleCommunityLanguageChanged = lang => {
-    this.setState({
-      defaultLanguage: lang.value,
-      languages: [lang.value]
-    })
+    this.setState({ selectedDefaultLang: lang, selectedLanguages: [lang] })
```

Comment:
why `CommunityLanguageChanged`? I don't understand the reference to community

Reply:
community doesn't support multiple languages, so logic & ui is a bit different